site/content/en/docs/reference/kubectl-kueue.md

```diff
@@ -75,17 +75,166 @@ You can run `kubectl kueue help` in the terminal to get the full list of command
 
 The following table includes short descriptions and the general syntax for all of the kueuectl operations:
 
-| Operation | Syntax                       | Description                                             |
-|--------|------------------------------|---------------------------------------------------------|
-| create | kubectl kueue create [TYPE] [NAME] [flags] | Create resource. | 
+| Operation | Syntax                                     | Description          |
+|-----------|--------------------------------------------|----------------------|
+| create    | kubectl kueue create [TYPE] [NAME] [flags] | Create a resource.   | 
+| resume    | kubectl kueue resume [TYPE] [NAME] [flags] | Resume the resource. | 
+| stop      | kubectl kueue stop [TYPE] [NAME] [flags]   | Stop the resource.   | 
 
 
-## Examples: Common operations
+## Resource types
 
-```shell
-# Create a local queue 
+The following table includes a list of all the supported resource types and their abbreviated aliases:
+
+| Name       | Short | API version            | Namespaced | Kind       |
+|------------|-------|------------------------|------------|------------|
+| localqueue | lq    | kueue.x-k8s.io/v1beta1 | true       | LocalQueue |
+| workload   | wl    | kueue.x-k8s.io/v1beta1 | true       | Workload   |
+
+
+## Create
```

Comment:
I don't necessarily oppose all these details, but I think we can get away without them.

Using `help` in the CLI should be more than enough.

Reply:
I'm agree with you. @mwielgus, wdyt?